packages/ckeditor5-restricted-editing/docs/_snippets/features/restricted-editing.js

```diff
@@ -75,7 +75,7 @@ async function reloadEditor( config ) {
 			window.editor.ui.view.toolbar,
 			item => item.label && [ 'Enable editing', 'Disable editing' ].includes( item.label )
 		),
-		text: 'Click to switch restricted editing mode.',
+		text: 'Click to add new editable region on selection or remove an existing editable region.',
```

Comment:
Technically makes sense but it's a lot of text

![](https://user-images.githubusercontent.com/1099479/111620037-85166f80-87e6-11eb-94c2-fbaff5db9350.png)

how about

<img width="745" alt="Screenshot 2021-03-18 at 12 35 42" src="https://user-images.githubusercontent.com/1099479/111620103-93648b80-87e6-11eb-83dd-6a70e24acf93.png">

?

Reply:
Sure, looks better.

Reply:
Fixed in commit.